File: sources/academy/webscraping/scraping_basics_javascript/exercises/f1academy_drivers.mjs
import * as cheerio from 'cheerio';

const url = 'https://www.f1academy.com/Racing-Series/Teams';
const response = await fetch(url);

if (!response.ok) {
  throw new Error(`HTTP ${response.status}`);
}

const html = await response.text();
const $ = cheerio.load(html);

console.log($('.driver').length);

File: sources/academy/webscraping/scraping_basics_python/exercises/f1academy_drivers.py
import httpx
from bs4 import BeautifulSoup

url = "https://www.f1academy.com/Racing-Series/Teams"
response = httpx.get(url)
response.raise_for_status()

soup = BeautifulSoup(response.text, "html.parser")
print(len(soup.select('.driver')))
